client/src/pages/sniffers/SniffersPage/Sniffer.tsx

```diff
@@ -10,13 +10,14 @@ import "reactflow/dist/style.css";
 import { VscTypeHierarchy } from "react-icons/vsc";
 
 const SnifferNode = (props: NodeProps) => {
+  const truncatedAddress = `${props.data.address.substring(0, 40)}...`;
```

Comment:
Please use css with elipsis and max-width and not substring. 
Thank you very much!

Reply:
ok

Reply:
Hey thank you very much Manmit :)
Approved